src/quic/node_quic_session.h

```diff
@@ -625,8 +625,8 @@ class QuicApplication : public MemoryRetainer {
 // control. In other words, there's quite a bit going on within
 // a QuicSession object.
 class QuicSession : public AsyncWrap,
-                    public mem::NgLibMemoryManager<QuicSession, ngtcp2_mem>,
-                    public StatsBase<QuicSessionStatsTraits> {
+                    public StatsBase<QuicSessionStatsTraits>,
+                    public mem::NgLibMemoryManager<QuicSession, ngtcp2_mem> {
```

Comment:
Does this change anything? If it does, that’s probably a sign of (another?) bug

Reply:
oh, that was a leftover. I was trying it just in case it did.